src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/controller/PrisonConfigController.kt

```diff
@@ -61,4 +64,40 @@ class PrisonConfigController(
   fun getSupportedPrisons(): List<String> {
     return prisonConfigService.getSupportedPrisons()
   }
+
+  @PreAuthorize("hasRole('VISIT_SCHEDULER')")
```

Comment:
This is nothing to do with the timetable change - why is it in this PR?

Reply:
covers VB-2066, VB-2209 and VB-2151

Reply:
Which JIRA story covers the "exclude a date" feature?

are feature deployment and prison configuration being conflated?